common-custom-user-data-maven-extension/src/main/resources/gradle-enterprise.sample.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+
+<!-- Minimal gradle-enterprise.xml configuration file for Maven.
+
+     This file specifies only the URL of the Gradle Enterprise server, assuming that the rest of the Gradle Enterprise
+     configuration is provided programmatically in the extension.
+-->
```

Comment:
Please add a comment reminding people to rename this file in order for it to take affect.